.github/configuration/upgrade-test-configuration.yml

```diff
@@ -86,7 +86,7 @@ upgrade-version: [[
 ],
 [
   {
-    version: 15.2,
+    version: 15.3,
```

Comment:
Let's keep the 15.2 -> target.latest upgrade path and add new entry for 15.3 -> target.latest so that we have atleast one upgrade path which upgrades from lowest minor to this latest minor.  
Also, let's update the aboe entry 14.8 -> target.latest to 14.9 -> target.latest